Cargo.lock

```diff
@@ -155,6 +161,18 @@ version = "0.3.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "e3b5ca7a04898ad4bcd41c90c5285445ff5b791899bb1b0abdd2a2aa791211d7"
 
+[[package]]
+name = "bytecount"
+version = "0.3.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f861d9ce359f56dbcb6e0c2a1cb84e52ad732cadb57b806adeb3c7668caccbd8"
+
+[[package]]
+name = "bytecount"
+version = "0.6.0"
```

Comment:
2 different versions of `bytecount`? :(

Reply:
Not sure why, but nom-tracable depends explicitely on both nom_locate-1.0.0 and nom_locate-2.0.0, and each depends on its own version of bytecount..